lib/motions/open-file-under-cursor.coffee

```diff
@@ -0,0 +1,58 @@
+{Motion} = require './general-motions'
+path = require 'path'
+{Directory} = require 'atom'
+shell = require 'shell'
+{exec} = require 'child_process'
+utils = require '../utils'
+
+class OpenFileUnderCursor extends Motion
+  execute: (count) ->
+    ###
+      /usr/bin/test
+      ../../.travis.yml
+      ./search-motion
+      ./index
+      http://github.com
+      https://thing.com/item%20test
+      //github.com
+    ###
+    wordRegex = /[a-z0-9\.\-_\/\\%:]+/i
+    @editor.getCursors().forEach (cursor) =>
+      currentWord = utils.getCurrentWord @editor, cursor, wordRegex
+      # range = cursor.getCurrentWordBufferRange wordRegex: wordRegex
+      # selectedPath = @editor.getTextInRange range
+      range = currentWord.range
+      selectedPath = currentWord.word
```

Comment:
oop, you're right.  renaming word to selectedPath takes another line anyway so this change wouldn't buy much.  Never mind!